x-pack/platform/plugins/shared/stack_connectors/server/routes/get_well_known_email_service.test.ts

```diff
@@ -80,4 +81,28 @@ describe('getWellKnownEmailServiceRoute', () => {
       body: {},
     });
   });
+
+  it('returns aws simple email service (ses) config if set', async () => {
```

Comment:
nit: Should we add a test that tests when the config is not set and the service is `ses`?

Reply:
But that shouldn't happen, we have the default ses config when no kibana config is set